baremetal/metal3machine_manager_test.go

```diff
@@ -1664,6 +1681,17 @@ var _ = Describe("Metal3Machine manager", func() {
 			ExpectSecretDeleted:       true,
 			ExpectClusterLabelDeleted: false,
 		}),
+		Entry("BMH NetworkData and MetaData should not be cleaned on deprovisioning", testCaseDelete{
```

Comment:
maybe UserData too, as you have added it in the `bmhSpecSomeImg`?

Reply:
Fixed! Thanks